### PR TITLE
Tweak Code Challenge for New Position

### PR DESCRIPTION
**Pull request by tkburns**
### Summary

Update the Code Challenge Instructions for the new position (and also fix the Lifeway logo).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,10 @@ Use whatever languages or tech you want -- we are looking for clean, testable co
 
 You will get bonus points if...
 
-* Basic react.js understanding is demonstrated...
-* A basic "gateway" api is created to consolidate and proxy calls to SWAPI...
-* You add cool and creative features not included in the spec...
+* A decent understanding of React is demonstrated
```
Comment (daynewright) [inline README.md:30]: What about a comment on testing?
```suggestion
* A decent understanding of React is demonstrated
* A decent understanding of front-end testing is demonstrated
```

**Review by daynewright — APPROVED** [review]
Looks good! 🔥

**Comment by charlesbthomas** [comment]
derp

**Comment by tkburns** [comment]
A wild Charlie appears!

**Review by daynewright — APPROVED** [review]

**Pull request merged**